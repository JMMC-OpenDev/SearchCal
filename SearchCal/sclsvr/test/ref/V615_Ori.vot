version https://git-lfs.github.com/spec/v1
oid sha256:fb4c8b304909eaaa89b31df4ec9ed1ae66a1edae8606307237274aa36e172ba9
size 210964
