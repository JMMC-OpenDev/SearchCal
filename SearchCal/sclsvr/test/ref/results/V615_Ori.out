version https://git-lfs.github.com/spec/v1
oid sha256:1d414e063520cfdfc6ab796d9a04c310b4e1c45894f906c1b86cd47f9c99beb5
size 359251
